test/harmony-regression/version.txt

```diff
@@ -1 +1 @@
-0.1.3
```

Comment:
I think you updated every version, but we should only update those that you changed here, unless the notebooks were changed before and the versions not updated.

Reply:
From that I think, we should leave the original versions on harmony-regression, hoss, hybig, swath-projector, and variable-subsetter as they were?

Reply:
Oh, right. So in DAS-1867, I added/updated that `Prerequisite` cell block in 9/10 of the notebooks, and of those 9 I apparently only updated the trajectory-subsetter version. 

Also from checking just now, harmony-regression had no change in DAS-2050 and DAS-1867, so that version doesn't need to be increased.

Reply:
Gotcha, These notebook changes are only for the ones that had "Code" instead of "markdown" cell types.  I can approve.